test/test-connector-auth.js

```diff
@@ -102,7 +107,7 @@ describe('OpenAPI connector - security', function() {
   });
 });
 
-function givenDataSource(spec, authz, done) {
+async function givenDataSource(spec, authz, done) {
```

Comment:
A callback-based function returning a promise? That looks suspicious to me. What are you trying to achieve by this change to `async function`? Did you perhaps want to rewrite this helper in a similar style as you did in `test/test-connector-swagger2.js`?

Reply:
Fixed.